### PR TITLE
Implementing pre-commit.ci and blacken-docs hook

### LINKED ISSUE #98: Add blacken-docs to pre-commit
### Description of the feature

Something I discovered while working on #64 was that some of the code snippets were not compliant with the code style guidelines. 

After looking for a solution, I came across [blacken-docs](https://github.com/asottile/blacken-docs/), which turned out to be extremely useful.

### Steps for implementing the feature

The main idea behind this new feature would be to add this doc style checker to our current style tests via pre-commit.

### Useful links and references

Blacken-docs: https://github.com/asottile/blacken-docs/

### PR DESCRIPTION
**Pull request by RobPasMue**
Closes #98

### CODE REVIEW

File: doc/source/coding-style/pep8.rst [hunk 1]
```diff
@@ -797,26 +799,26 @@ example, to extract all consonants in a sentence:
...
+    vowels = 'aeiou'
+    consonants = []
+    for letter in sentence:
+        if letter.isalpha() and letter.lower() not in vowels:
+            consonants.append(letter)
+
+    ode-block:: pycon 
+
+    >>> print(f"{consonants = }")
+    consonants = ['T', 'h', 's', 's', 's', 'm', 'p', 'l', 's', 'n', 't', 'n', 'c']
```
Comment (RobPasMue) [inline doc/source/coding-style/pep8.rst:822]: Shii*... What happened here? Let me revert that

File: doc/source/how-to/testing.rst [hunk 2]
```diff
@@ -190,29 +190,26 @@ file would be:
...
-                    Deserialized Numpy array.
-        
-                """
-                arrays = []
-                for chunk in chunks:
-                    arrays.append(
-                        np.frombuffer(chunk.payload, ANSYS_VALUE_TYPE[chunk.value_type])
-                    )
+           def parse_chunks(chunks):
+               """Deserialize gRPC chunks into a Numpy array.
```
Comment (germa89) [inline doc/source/how-to/testing.rst:194]: No empty line after the ``..code-block``?

**Review by germa89 — APPROVED** [review]
LGTM.

File: doc/source/how-to/testing.rst (hunk 2 above)
Comment (RobPasMue) [reply doc/source/how-to/testing.rst:195]: Totally right - must be a bug in ``blacken-docs``

**Pull request merged**